### ISSUE TITLE
Question about few-shot learning

### ISSUE THREAD

**Issue by flying-cmd** [Hank0626/CALF#11]
Hi,
I have a question for your few-shot learning setup. Suppose I want to run a few‑shot learning task on dataset  ETTh1. Should I (i) train the model using only the first 10 % of ETTh1’s training set and then evaluate on ETTh1’s test set, or (ii) first train the model on a different dataset (e.g., dataset weather), then fine‑tune it using the first 10 % of ETTh1’s training set, and finally evaluate on ETTh1’s test set?

**Comment by Hank0626** [comment]
> Hi, I have a question for your few-shot learning setup. Suppose I want to run a few‑shot learning task on dataset  ETTh1. Should I (i) train the model using only the first 10 % of ETTh1’s training set and then evaluate on ETTh1’s test set, or (ii) first train the model on a different dataset (e.g., dataset weather), then fine‑tune it using the first 10 % of ETTh1’s training set, and finally evaluate on ETTh1’s test set?

It's the first one: train on the first 10% of ETTh1's training set, then evaluate on ETTh1's test set.

**Comment by flying-cmd** [comment]
> > Hi, I have a question for your few-shot learning setup. Suppose I want to run a few‑shot learning task on dataset  ETTh1. Should I (i) train the model using only the first 10 % of ETTh1’s training set and then evaluate on ETTh1’s test set, or (ii) first train the model on a different dataset (e.g., dataset weather), then fine‑tune it using the first 10 % of ETTh1’s training set, and finally evaluate on ETTh1’s test set?
> 
> It's the first one: train on the first 10% of ETTh1's training set, then evaluate on ETTh1's test set.

Thanks for your reply!
